src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java

```diff
@@ -0,0 +1,396 @@
+package uk.gov.hmcts.futurehearings.hmi.unit.testing.testsuites;
+
+import com.aventstack.extentreports.ExtentTest;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.*;
```

Comment:
Remove Wildcards from imports and make it class specific please...

Reply:
Expanded the import statements as part of the commit below